nnvm/python/nnvm/compiler/build_module.py

```diff
@@ -211,11 +211,7 @@ def optimize(graph, shape, dtype="float32", layout=None, target=None):
         graph = graph.apply(["CorrectLayout"])
 
     if cfg.ext_accel and cfg.ext_accel == 'tensorrt':
-        if isinstance(target, dict):
```

Comment:
I think you still need to keep this logic to avoid partitioning graphs in PrecomputePrune pass where target is llvm.

Reply:
Thanks for reviewing! Could you explain this more? The ext_accel would still have to be set to tensorrt for partitioning to happen. PrecomputePrune doesn't seem to call the optimize function either. Why does it matter what the target is if the user has specified ext_accel=tensorrt?
Thanks!

Reply:
@reminisce